src/bin/nydus-image/main.rs

```diff
@@ -1804,3 +1805,12 @@ impl Command {
         nydus_service::validate_threads_configuration(v).map(|s| s.to_string())
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::Command;
+    #[test]
+    fn test_ensure_file() {
+        Command::ensure_file("/proc/self/fd/0").unwrap();
```

Comment:
Maybe use `/dev/stdin`? It seems more compatible with macos. (the unit tests will run in macos env in release action).

Reply:
fixed

Reply:
https://github.com/zyfjeff/image-service/actions/runs/5997383931/job/16263668460

release action ci ran successfully 

ping @imeoer

Reply:
Thanks for the testing!